src/main/java/uk/ac/cam/cl/dtg/segue/api/SegueContentFacade.java

```diff
@@ -54,7 +54,7 @@
 /**
  * Segue Content Facade
  * <br>
- * This class specifically caters for the Rutherford physics server and is expected to provide extended functionality to
+ * This class specifically caters for the Rutherford Computer Science server and is expected to provide extended functionality to
```

Comment:
That sounds like the entire class needs re-examining and potentially getting rid of, if it's to cater for a particular server?!

Reply:
This class is currently in use.

Reply:
The description dates back to 2015, according to the blame, so it might be that that it was written with the infrastructure of the time in mind and didn't really consider alternatives? Isaac isn't running on UoC's server(s) any more but everything seems to work fine so this may simply need a larger update than replacing the 'Physics' bit?